devito/operator/operator.py

```diff
@@ -828,28 +829,14 @@ def __getnewargs_ex__(self):
         return (None,), {}
 
     def __setstate__(self, state):
-        soname = state.pop('_soname', None)
+        soname = state.pop('soname', None)
         binary = state.pop('binary', None)
         for k, v in state.items():
             setattr(self, k, v)
-        # If the `sonames` don't match, there *might* be a hidden bug as the
```

Comment:
So that's not an issue anymore?

Reply:
That wasn't really an issue strictly speaking -- more of a sanity check upon unpickling aimed to spot bugs somewhere else. Which has non negligible overhead

Reply:
Ok. Is there a test for it?

Reply:
yes, pickling tests for _all_ (many are picked recursively) Devito objects are available in `test_pickle.py`.